service/service.go

```diff
@@ -78,6 +80,22 @@ func New(config Config) (*Service, error) {
 
 	var err error
 
+	features := make(map[string]bool)
```

Comment:
Rudimentary mechanism for feature flags. It would allow us to have MachinePools, but disable it if we need to make a new release.